docs/examples/adapters/http_npm_download_counts.juttle

```diff
@@ -0,0 +1,13 @@
+/*
+ * Query daily download counts from npm
+ *
+ * Input parameters:
+ *
+ *  package: package name (i.e. 'juttle')
+ */
+
+input package: text -default 'juttle';
+
+read http -url 'https://api.npmjs.org/downloads/range/last-month/${package}'
+     -rootPath 'downloads' -timeField 'day'
```

Comment:
When we were looking at the output earlier, you noticed a bug where the day was sticking around as a column. Did that get fixed?